VirtoCommerce.Platform.Core/Notifications/PlatformEmailSender.cs

```diff
@@ -0,0 +1,14 @@
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace VirtoCommerce.Platform.Core.Notifications
+{
+    public class PlatformEmailSender : IEmailSender
+    {
+        public Task SendEmailAsync(string to, string type, string languageCode, Dictionary<string, object> parameters)
```

Comment:
Use this signature  `public Task SendEmailAsync(string email, string subject, string message)`